src/components/HandsonTable/HandsonTable.styles.js

```diff
@@ -21,8 +21,9 @@ export const StyledHotTable = styled(HotTable)`
   .ht_clone_top_left_corner {
     z-index: 1;
   }
-  
+
   .handsontableInput {
     box-sizing: content-box;
   }
+  /* stylelint-enable selector-class-pattern */
```

Comment:
no need to enable it in the end of the file, it's fine to have only /* stylelint-disable selector-class-pattern */ at the beginning